man/openqa-label-all.1

```diff
@@ -0,0 +1,65 @@
+.\"                                      Hey, EMACS: -*- nroff -*-
```

Comment:
This file seems to contain a verbatim copy of the CLI argument descriptions of the actual script. This is not great as it might become out of sync with its source of truth while at the same time not providing much more value then the `--help` text itself. So I would rather avoid having this file.

The proper way of having a manpage for this script would probably be via [argparse-manpage](https://pypi.org/project/argparse-manpage/). (I haven't used this myself, though. So I'm not sure whether it is any good; it is just the first result my brief search revealed.)

Reply:
That's a good idea :-)

I've done that, although it seems to have rather fixed ideas about the author line at the end, which I've forced to produce something that doesn't just credit one author, which seemed like the right thing to do in this case.

I've guessed at the way one is supposed to declare the new dependency on python3-argparse-manpage -- hopefully that works on openSUSE (we have our own way of doing that in `debian/control` so I've not tested that bit locally.  I guess the CI will tell me if I did the right thing)

Reply:
Thanks